backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/profiles/DiskProfileHelper.java

```diff
@@ -149,4 +153,9 @@ private boolean isDiskProfilePermitted(DiskProfile diskProfile,
                         diskProfile.getId(),
                         VdcObjectType.DiskProfile) != null;
     }
+
+    private boolean shouldSkipDiskProfilesValidation(DiskImage diskImage) {
```

Comment:
it's not only about the validation anymore - so I'd rename this method to shouldSetDiskProfileOnImage or something like that